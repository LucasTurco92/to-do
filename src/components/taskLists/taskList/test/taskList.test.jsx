import { render } from '@testing-library/react';
import TaskList from '../taskList';

it('When it invokes the TaskList component, should render it properly', ()=>{

    const { container } = render(<TaskList />);
    
    expect(container).toMatchSnapshot();
});