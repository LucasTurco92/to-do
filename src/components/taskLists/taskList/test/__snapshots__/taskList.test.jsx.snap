// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When it invokes the TaskList component, should render it properly 1`] = `
<div>
  <div
    class="sc-bdvvtL iIcWDR"
  >
    <div
      class="sc-gsDKAQ bKQTPl"
    />
  </div>
</div>
`;
